index.css

```diff
@@ -0,0 +1,96 @@
+.Main-Page {
+  max-width: 500px;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 590px;
+  width: 375px;
+  left: 0;
+  top: 92px;
+  border-radius: 0;
+  border: 2px solid black;
+  padding: 120px, 24px, 120px, 24px;
+}
```

Comment:
- [ ] Your body does not have the correct font styling as in the Figma design, kindly add the correct font families to all your elements. You may want to use [Google Fonts](https://fonts.google.com/) to import the fonts you need.
- [ ] Your body has a default margin, so your content does not take up the entire screen. Kindly fix this issue.
![image](https://user-images.githubusercontent.com/90556221/176879072-6e812056-ae8c-40aa-982f-1ec7dd971837.png)